src/components/modals/email-confirmation-modal.style.js

```diff
@@ -0,0 +1,19 @@
+export const style = {
+  box: {
+    margin: { xs: '0 auto', sm: 0 },
+    padding: { xs: 0, sm: 2, md: 7 },
+    textAlign: 'center'
+  },
+  button: {
+    marginTop: '30px',
+    backgroundColor: '#262738',
```

Comment:
use colors from palette

Reply:
but what if color from figma #262738 does not presented in palette ?